ui/pages/confirmations/components/confirm/row/dataTree.tsx

```diff
@@ -34,7 +34,7 @@ export const DataTree = ({ data }: { data: Record<string, TreeData> }) => (
           }) ? (
             <ConfirmInfoRowAddress address={value} />
           ) : (
-            <ConfirmInfoRowText text={value} />
+            <ConfirmInfoRowText text={sanitizeString(value)} />
```

Comment:
I need to do some fixes in this component to be able to unit test cover it. It will be in a followup PR for this issue: https://app.zenhub.com/workspaces/-confirmations-secure-ux-6245e6e2348677001213b8d2/issues/gh/metamask/metamask-extension/23689